tests/POW/test_POW.cpp

```diff
@@ -792,17 +803,28 @@ BOOST_AUTO_TEST_CASE(opencl_gpu_mining_and_verification_1)
     BOOST_REQUIRE(!verifyWinningNonce);
 }
 
-// Please enable the OPENCL_GPU_MINE option in constants.xml to run this test case
-BOOST_AUTO_TEST_CASE(opencl_gpu_mining_and_verification_2)
+// Please enable the OPENCL_GPU_MINE or CUDA_GPU_MINE option in constants.xml to run this test case
+BOOST_AUTO_TEST_CASE(gpu_mining_and_verification_2)
 {
-    if (!OPENCL_GPU_MINE)
+    if (!OPENCL_GPU_MINE && !CUDA_GPU_MINE)
     {
-        std::cout << "OPENCL_GPU_MINE option is not enabled, skip test case "
-                     "opencl_gpu_mining_and_verification_2"
+        std::cout << "OPENCL_GPU_MINE and CUDA_GPU_MINE option are not "
+                     "enabled, skip test case "
+                     "gpu_mining_and_verification_1"
```

Comment:
I guess this should be 2 instead of 1, right?

Reply:
Yes, you are correct.